src/amo/components/CollectionList/index.js

```diff
@@ -51,6 +51,9 @@ export class CollectionListBase extends React.Component<Props> {
           <Card className="CollectionList-create">
             {this.renderManager()}
           </Card>
+          <p className="CollectionList-placeholder">
+            Please save your collection and then you can add add-ons to it
```

Comment:
We need to make sure that all strings that are displayed are localizable, and to do that we need to wrap it in an `i18n` function. In this case we can use `i18n.gettext`. You can see an example of this in the same file at line 38 where we have `logInText={i18n.gettext('Log in to create a collection')}`.

Reply:
ah! got it - thanks